pkg/config/defaults.go

```diff
@@ -134,17 +155,10 @@ const (
 	UIEnabledKey = "ui.enabled"
 )
 
-func setDefaultLocalConfig() {
-	viper.SetDefault(DatabaseTypeKey, DefaultDatabaseType)
-	viper.SetDefault(DatabaseKVLocalPath, DefaultDatabaseLocalKVPath)
-	viper.SetDefault(BlockstoreLocalPathKey, DefaultBlockstoreLocalPath)
-	viper.SetDefault(AuthEncryptSecretKey, DefaultAuthEncryptSecretKey)
-	viper.SetDefault(BlockstoreTypeKey, DefaultBlockstoreType)
-}
-
 func setDefaults(local bool) {
 	if local {
-		setDefaultLocalConfig()
+		viper.SetDefault(DatabaseTypeKey, DefaultDatabaseType)
+		viper.SetDefault(AuthEncryptSecretKey, DefaultAuthEncryptSecretKey)
```

Comment:
```suggestion
		viper.SetDefault(DatabaseTypeKey, LocalDefaultDatabaseType)
		viper.SetDefault(AuthEncryptSecretKey, LocalDefaultAuthEncryptSecretKey)
```
Just a suggestion, It was a bit hard for me to understand, 1. if they are defaults why set them only if local? 2. why are the other local defaults always set?

Reply:
Maybe even comment in the definition of these values are defaults for the case of --local-settings.

Reply:
- About the names of the constants - I kept the naming pattern.
- About the defaults - we are talking about the storage type?

Reply:
I've pushed another commit - removed the local-settings flag.